app/models/date_range.rb

```diff
@@ -0,0 +1,30 @@
+class DateRange
+  attr_reader :start_date, :end_date
```

Comment:
What's the reason these attributes are public? The `DateRange` class is a simple Service object or "View Helper", I don't see the necessity for these to ever be called outside the scope of this class. So I'd suggest to move them to private:

```ruby

private

attr_reader :start_date, :end_date
```